tuning/data/data_handlers.py

```diff
@@ -38,8 +38,11 @@ def tokenize_and_apply_input_masking(
     # TODO: Eventually move the code here
     combined = combine_sequence(input, output, eos_token=tokenizer.eos_token)
 
-    tokenized_comb_seqs = tokenizer(combined, **tokenizer_kwargs)
-    tokenized_input = tokenizer(input, **tokenizer_kwargs)
+    fn_kwargs = tokenizer_kwargs.get("fn_kwargs", {})
+    tokenizer_inner_kwargs = fn_kwargs.get("tokenizer_kwargs", {})
+
+    tokenized_comb_seqs = tokenizer(combined, **tokenizer_inner_kwargs)
+    tokenized_input = tokenizer(input, **tokenizer_inner_kwargs)
```

Comment:
Reference of discussion for this change: https://github.com/foundation-model-stack/fms-hf-tuning/pull/381#discussion_r1851174125

Reply:
Thanks @Abhishek-TAMU